### ISSUE TITLE
Why the result is different from local run of the repo and the web demo version?

### ISSUE THREAD

**Issue by ShroukMansour** [clovaai/CRAFT-pytorch#53]
I tested my images first on the web demo, but when I test using the repo locally on my machine I get different results. Is there a difference in the parameters or you are using another weight. I need to know the difference to apply it to my local version.

Website demo result that recognizes line by line[what I need] 
![Screenshot from 2019-10-02 14-40-36](https://user-images.githubusercontent.com/18499889/66044856-c2b80780-e522-11e9-956d-7dc79e2d95fa.png)

Local repo result that recognizes word by word

![res_4558](https://user-images.githubusercontent.com/18499889/66044723-82588980-e522-11e9-9123-4e1d69662110.jpg)

**Comment by cong235** [comment]
Did you used refiner_model ?

**Comment by ShroukMansour** [comment]
I've used it but it still didn't recognize the whole line it leaves some words at the start and at the end!!

**Comment by ShroukMansour** [comment]
@ClovaAIAdmin  @YoungminBaek  @cong235 If you can help, please!

**Comment by cong235** [comment]
please confirm your setting --refine=True. I try to test with your image. It still can detect by line. Or another way, you can use rule base for detect box in 1 line.
![res_66044856-c2b80780-e522-11e9-956d-7dc79e2d95fa](https://user-images.githubusercontent.com/16643244/66296649-3ccc0000-e918-11e9-90da-355d30ae38b0.jpg)

**Comment by ShroukMansour** [comment]
Would you illistrate more the another option please. And try  the original image and tell me the thresholds values you are using @cong235 
![4558](https://user-images.githubusercontent.com/18499889/66297317-be5b6800-e8ef-11e9-81b4-7b61d530c409.png)

**Comment by cong235** [comment]
it still working good, but this is pretrain so the result will not good like product version in the website. And in the paper, author said about some language their method work too bad.

![res_xxsaxax](https://user-images.githubusercontent.com/16643244/66297630-3fc7f000-e91a-11e9-99c6-78d651d2ddc8.jpg)
<img width="892" alt="Screen Shot 2019-10-07 at 3 35 54 PM" src="https://user-images.githubusercontent.com/16643244/66297636-422a4a00-e91a-11e9-8030-428398122aab.png">

**Comment by ShroukMansour** [comment]
@cong235  this is was my main problem, the difference between the web demo version and the local repo version. Do you have an idea about changing some paramters to get something similar to the product version?

**Comment by cong235** [comment]
You can collect data and training with your data.
Follow this git.
https://github.com/backtime92/CRAFT-Reimplementation

**Comment by ShroukMansour** [comment]
I don't have a labeled dataset and it's hard to  find in Arabic

**Comment by cong235** [comment]
you should find it. google, facebook group..etc.

**Comment by ShroukMansour** [comment]
If you mean the data should look like an image with labeled boxes for lines, So I don't thnk something like exsits.

**Comment by ShroukMansour** [comment]
Tell me what was you thinkning about collecting it form google or facebook? @cong235

**Comment by cong235** [comment]
You can crawl data from google, then find 1 model detection and regconition for your lang. Export it to format can edit later. Then use your eyes to fix wrong data.

**Comment by thaihaquang** [comment]
> Tell me what was you thinkning about collecting it form google or facebook? @cong235

you can generate synthetic data like SynthText (mentioned in CRAFT paper) to train the model in your language, not only the real data from google or FB (of course, it is very good if you can collect then label the real data).
